src/main/java/com/laytonsmith/core/AliasCore.java

```diff
@@ -155,25 +140,26 @@ public boolean alias(String command, final MCCommandSender sender) {
 			Static.getLogger().log(Level.INFO, "Running alias on " + sender.getName() + " ---> " + command);
 		}
 
-		GlobalEnv gEnv = new GlobalEnv(parent.executionQueue,
-				MethodScriptFileLocations.getDefault().getConfigDirectory(), EnumSet.of(RuntimeMode.EMBEDDED));
-		StaticRuntimeEnv staticRuntimeEnv = new StaticRuntimeEnv(parent.profiler,
-				parent.persistenceNetwork, parent.profiles, new TaskManagerImpl());
-		CommandHelperEnvironment cEnv = new CommandHelperEnvironment();
-		cEnv.SetCommandSender(sender);
-		cEnv.SetCommand(command);
-		Environment env = Environment.createEnvironment(gEnv, staticRuntimeEnv, cEnv, compilerEnv);
+		Environment env;
+		try {
+			env = environment.clone();
+		} catch (CloneNotSupportedException ex) {
+			throw new Error(ex.getMessage());
```

Comment:
Assuming we don't expect this case to ever happen, it'd be best to throw `new Error(ex)` instead, passing the original stacktrace in the `caused by` exception stacktrace.